packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js

```diff
@@ -262,6 +263,14 @@ function verifyTypeScriptSetup() {
     );
   }
 
```

Comment:
Should also probably warn if a user for whatever reason decides to use more than one path (eg: `["components/*", "layouts/*"]`). What do you think?

Reply:
Thanks, I've just done this. I tried implementing multiple path support first but it seems like Webpack 4 does not support this. Webpack 5 does though so when CRA moves to that we can add this feature.

I've now created an `errors` array that I push strings into. I can't use the current `messages` array as they are just outputted as information that your `tsconfig.json` has been updated. If the `baseUrl` or `paths` are set up incorrectly I need to `process.exit(1)` as we shouldn't go any further.